index.html

```diff
@@ -12,7 +12,7 @@
 
     <script src="index.js"></script>
 
-    <h1>Hello World!</h1>
+    <h1>Hallo Welt!</h1>
```

Comment:
@aainn I don't understand german, is this right?

Reply:
jaja - alles richtig!